src/shared/types/mypage.ts

```
@@ -0,0 +1,71 @@
+  imageUrls: string[];
+  recruitmentEndDate: string;
+  recruitmentStartDate: string;
+  title: string;
+  id: number;
+}
+
+export interface ScrapCardProps {
+  cardInfo: ScrapCardItem[];
+}
+
+export interface SortOptions {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SortOptions'. Consider to delete it or rename it.

'SortOptions' is defined here:


(lint/suspicious/noRedeclare)